test/jasmine/assets/click.js

```diff
@@ -4,4 +4,4 @@ module.exports = function click(x, y) {
     mouseEvent('mousemove', x, y);
     mouseEvent('mousedown', x, y);
     mouseEvent('mouseup', x, y);
-};
\ No newline at end of file
+};
```

Comment:
@etpinard per my commit message... somehow eslint run through sublime cannot sort out the config files within the jasmine directory, even though as far as I can tell it's all correct. In any other directory things look fine. Have you seen this? If I change `test/jasmine/.eslintrc` from `"extends": "../../.eslintrc"` to `"extends": "../.eslintrc"` it no longer complains, though that's clearly not what you want as it'll think node is allowed.

I suppose we could make that change and also explicitly set `env.node=false`, that seems to work for me, but it's strange that this would be necessary.